src/components/Frontpage/index.js

```diff
@@ -14,241 +14,252 @@ import './FixMe.css'
-          {sidebar && sidebar.split('<hr>').map(__html =>
-            <div className="col-md-4 home_section" key={__html}>
-              <div className="home_section_icon">
-                <i className="fa" />
               </div>
-              <div dangerouslySetInnerHTML={{__html}} />
-            </div>
-          )}
+            }
+          </Calypso>
+          <Calypso type='event'>
+            {content =>
+              <div className={cx('news')}>
+                <Link
+                  to={lang === 'en' ? '/en/news?itemType=EVENT' : '/nyheter?itemType=EVENT'}
+                >
+                  <h2>
+                    <Translate>
+                      <English>Event</English>
+                      <Swedish>Event</Swedish>
+                    </Translate>
+                  </h2>
+                </Link>
+                <ul>
+                  {
+                    (content && content.length)
+                      ? content
+                        .filter((_, i) => i < 4)
+                        .map(item => <li key={item.id}>
+                          <Link
+                            to={lang === 'en' ? `/en/news/${item.id}` : `/nyheter/${item.id}`}
+                          >
+                            <h3>
+                              <Translate>
+                                <English>{item.titleEnglish}</English>
+                                <Swedish>{item.titleSwedish}</Swedish>
+                              </Translate>
+                            </h3>
+                          </Link>
+                          <span>
+                            {
+                              new Date(item.eventStartTime)
+                                .toLocaleDateString(
+                                  lang === 'en' ? 'en-US' : 'sv-SE',
+                                  {
+                                    day: 'numeric',
+                                    month: 'short',
+                                    year: 'numeric'
+                                  }
+                                )
+                            }
+                          </span>
+                          &bull;
+                          <span>
+                            {item.publishAsDisplay || item.authorDisplay}
+                          </span>
+                        </li>)
+                      : <h4 className={cx('empty')}>
+                        <Translate>
+                          <English>No upcoming events :(</English>
+                          <Swedish>Inga kommande event :(</Swedish>
+                        </Translate>
+                      </h4>
+                  }
+                </ul>
+                {
+                  content.length > 4 &&
+                  <div className="text-center">
+                    <Link
+                      to={lang === 'en' ? '/en/news?itemType=EVENT' : '/nyheter?itemType=EVENT'}
+                      className='more-btn'
```

Comment:
This needs to use the `cx`-function as earler, as the classes are scoped and therefore renamed.